ee-feature-pack/galleon-common/src/main/resources/packages/io.netty.netty-transport-native-epoll/pm/wildfly/tasks.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<tasks xmlns="urn:wildfly:wildfly-feature-pack-tasks:2.0">
+    <copy-artifact artifact="io.netty:netty-transport-native-epoll::linux-x86_64" to-location="modules/system/layers/base/io/netty/netty-transport-native-epoll/main/lib" extract="true" >
+        <filter pattern="META-INF/" include="true" />
+        <filter pattern="META-INF/native/*" include="true" />
+        <filter pattern="*" include="false"/>
+    </copy-artifact>
+<!--    <copy-artifact artifact="io.netty:netty-transport-native-epoll::linux-aarch_64" to-location="modules/system/layers/base/io/netty/netty-transport-native-epoll/main/lib" extract="true" >
+        <filter pattern="META-INF/" include="true" />
+        <filter pattern="META-INF/native/*" include="true" />
+        <filter pattern="*" include="false"/>
+    </copy-artifact>-->
```

Comment:
@ehsavoie Looks like this is pure cruft as it seems to be a commented out copy of the uncommented block.

Reply:
different classifier : linux-aarch_64